django-stubs/contrib/gis/geos/prototypes/predicates.pyi

```diff
@@ -22,6 +22,7 @@ geos_crosses: Any
 geos_disjoint: Any
 geos_equals: Any
 geos_equalsexact: Any
+geos_equalsidentical: Any
```

Comment:
`BinaryPredicate`? https://github.com/django/django/blob/5f3cdf219de71021cc5b965695dbe0c881c0a4f1/django/contrib/gis/geos/prototypes/predicates.py#L42

You can type other values here as well.